src/txs/Tx.tsx

```diff
@@ -25,23 +25,25 @@ import { isBroadcastingState, latestTxState } from "data/queries/tx"
 import { useIsWalletEmpty } from "data/queries/bank"
 
 import { Pre } from "components/general"
-import { Flex, Grid } from "components/layout"
-import { FormError, Submit, Select, Input, FormItem } from "components/form"
+import { Grid, Flex } from "components/layout"
+import { FormError, Select, Input, FormItem } from "components/form"
 import { Modal } from "components/feedback"
 import { Details } from "components/display"
 import { Read } from "components/token"
 import ConnectWallet from "app/sections/ConnectWallet"
 import useToPostMultisigTx from "pages/multisig/utils/useToPostMultisigTx"
 import { isWallet, useAuth } from "auth"
 import { PasswordError } from "auth/scripts/keystore"
-
+import { Submit } from "components/form"
```

Comment:
Any reason this was taken from the grouped imports from `components/form`?

Reply:
Was related to testing. Reverted now.